bf_simple_yk.c

```diff
@@ -103,12 +105,15 @@ int main(int argc, char *argv[]) {
     if (cells == NULL)
         err(1, "out of memory");
 
+    YkMT *mt = yk_mt_new();
+    yk_mt_hot_threshold_set(mt, 5);
```

Comment:
Do we need to change the threshold?

Reply:
Probably not actually. Will revert.

Reply:
Fixed in [6522d09](https://github.com/ykjit/ykcbf/pull/10/commits/6522d09fd5d247f09605de37bcaab82324d845ad)